app/lib/filterServices.js

```diff
@@ -38,6 +38,7 @@ function filterServices(results, limits) {
 
     item.openingTimesMessage = openingTimesMessage;
     item.isOpen = isOpen;
+    item.distanceInMiles = item.dist;
```

Comment:
This doesn't need to be done if the query is changed to `distanceField: 'distanceInMiles'`
https://github.com/nhsuk/nearby-services-api/pull/30/files#diff-6b3c91a372d0bb755710f47f6b10c6e3R20